packages/mem_usage/mem_usage.0.1.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+synopsis: "Cross-platform stats about memory usage"
+maintainer: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+authors: "Romain Beauxis <toots@rastageeks.org>"
+license: "GPL-2.0"
```

Comment:
Is it
```suggestion
license: "GPL-2.0-only"
```
or
```suggestion
license: "GPL-2.0-or-later"
```
?
I did not find it in the repo

Reply:
It is not.. 😆 Let's keep the default? This is a really small package, I should perhaps have made it BSD or even less.. Might do it for the next release.

Reply:
Ok, I'll merge, but let's fix the license in the next release :)